index.html

```diff
@@ -1,5 +1,5 @@
 <!DOCTYPE html>
-<html lang="en">
+<html lang="en" xmlns:th="http://www.thymeleaf.org">
```

Comment:
Added thymeleaf

Source -(https://www.tutorialspoint.com/java/index.htm)

Reply:
Are you using any thymeleaf components?  If not, this import is not needed.